### PR TITLE
Symfony 6

### PR DESCRIPTION
**Pull request by jmsche**
Hi,

This little PR to allow Symfony 6 :)

I adapted the CI script so it would run on all supported major versions of Symfony.

### CODE REVIEW

**Comment by lyrixx** [comment]
Thanks a lot for this PR.

Unfortunately, it looks like some upstream vendor should be updated first.

**Comment by jmsche** [comment]
Hi Grégoire,

Yes I noticed the same. 

I opened an issue on the vendor, I'll see if I can do some PRs there as well but I'm not sure I'll be able to achieve this as it seems to require a lot of work, on a vendor I don't know well :)

**Comment by Korbeil** [comment]
Hey, I updated Jane, could you reload the tests / push a build ?

**Comment by Korbeil** [comment]
You have to use Jane ^7 to have the Symfony 6 compatible stuff. This should be an easy migration, if you want I can do it in another PR to avoid conflicting with your changes.

**Comment by jmsche** [comment]
Yes, if you can do that in an other PR it would be awesome :)

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -40,18 +40,20 @@ jobs:
...
             fail-fast: false
             matrix:
                 php-version: ['7.4', '8.0']
+                symfony-version: ['4.4.*', '5.3.*']
                 composer-flags: ['']
                 name: ['']
                 include:
-                    - php-version: 7.4
-                      composer-flags: '--prefer-lowest'
-                      name: '(prefer lowest dependencies)'
```
Comment (lyrixx) [inline .github/workflows/ci.yml:54]: Why did you remove this test ? it's supper important for us to ensure BC with older SF version

**Review by Korbeil — APPROVED** [review]
Thanks a lot for this contribution :)

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -70,8 +72,11 @@ jobs:
               with:
                   php-version: ${{ matrix.php-version }}
 
+            - name: Install Symfony Flex
+              run: composer global require --no-progress --no-scripts --no-plugins symfony/flex
```
Comment (lyrixx) [inline .github/workflows/ci.yml:76]: why do you need flex?

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -40,18 +40,20 @@ jobs:
...
                 php-version: ['7.4', '8.0']
+                symfony-version: ['4.4.*', '5.3.*']
                 composer-flags: ['']
                 name: ['']
                 include:
-                    - php-version: 7.4
-                      composer-flags: '--prefer-lowest'
-                      name: '(prefer lowest dependencies)'
+                    - php-version: '8.0'
+                      symfony-version: '6.0.*'
```
Comment (lyrixx) [inline .github/workflows/ci.yml:56]: It looks like you don't use it

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -70,8 +72,11 @@ jobs:
               with:
                   php-version: ${{ matrix.php-version }}
 
+            - name: Install Symfony Flex
+              run: composer global require --no-progress --no-scripts --no-plugins symfony/flex
+
             - name: Install Composer dependencies
-              run: composer update --prefer-dist --no-interaction --no-ansi ${{ matrix.composer-flags }}
+              run: composer update --prefer-dist --no-interaction --no-ansi
```
Comment (lyrixx) [inline .github/workflows/ci.yml:79]: Should be reverted

**Comment by jmsche** [comment]
Always happy to help :)

File: .github/workflows/ci.yml (hunk 2 above)
Comment (jmsche) [reply .github/workflows/ci.yml:76]: To enforce Symfony version

File: .github/workflows/ci.yml (hunk 3 above)
Comment (jmsche) [reply .github/workflows/ci.yml:56]: It is used, it's an extra run as I can't put Symfony 6.0 in the main matrix as it supports only PHP8.0+

File: .github/workflows/ci.yml (hunk 2 above)
Comment (lyrixx) [reply .github/workflows/ci.yml:79]: ok, I missed the line where you export `SYMFONY_REQUIRE` ; sorry !

File: .github/workflows/ci.yml (hunk 3 above)
Comment (lyrixx) [reply .github/workflows/ci.yml]: you are right, sorry, I missed the line where the export

File: .github/workflows/ci.yml [hunk 5]
```diff
@@ -40,18 +40,23 @@ jobs:
...
         runs-on: ubuntu-latest
+        env:
+            SYMFONY_REQUIRE: ${{ matrix.symfony-version }}
         strategy:
             fail-fast: false
             matrix:
                 php-version: ['7.4', '8.0']
+                symfony-version: ['4.4.*', '5.3.*']
                 composer-flags: ['']
                 name: ['']
```
Comment (lyrixx) [inline .github/workflows/ci.yml:53]: it could be removed, since it's not used anymore.

**Review by lyrixx — COMMENTED** [review]
Except one little comment, it's perfect 👍🏼 

Thanks a lot

File: .github/workflows/ci.yml (hunk 5 above)
Comment (jmsche) [reply .github/workflows/ci.yml:53]: What could be removed/is not used anymore?

Comment (lyrixx) [reply .github/workflows/ci.yml:53]: I fixed it in c10c5d68fe42f2d4aef295a255352f8d79eff085

Comment (jmsche) [reply .github/workflows/ci.yml:53]: My bad, I don't know why I thought your comment was about the highlighted line :D

Comment (lyrixx) [reply .github/workflows/ci.yml:53]: no problem :) again, thanks for the hard work here :) 💛
